omod/src/main/java/org/openmrs/module/reporting/web/datasets/LogicDataSetEditor.java

```diff
@@ -58,6 +65,10 @@ public void showDataset(ModelMap model, @RequestParam(required = false, value =
 		model.addAttribute("logicErrors", logicErrors);
 		model.addAttribute("tokens", tokens);
 		model.addAttribute("conceptNameTags", Context.getConceptService().getAllConceptNameTags());
+	}else {
+		
+		httpSession.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "reporting.logicModuleNotStarted.error");
+	}
 	}
```

Comment:
Also indent your code changes  appropriately following the java conventions.

Reply:
ok i have updated the Pr